### PR TITLE
matchmycolorLLC.Colibri version 22.2.0.7647

### PR DESCRIPTION
**Pull request by MichaelSchae**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92075)

### CODE REVIEW

**Comment by MichaelSchae** [comment]
> 

@microsoft-github-policy-service agree [company="matchmycolor LLC"]

**Comment by MichaelSchae** [comment]
@microsoft-github-policy-service agree company="matchmycolor LLC"

File: manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: matchmycolorLLC.Colibri
+PackageVersion: 22.2.0.7647
+PackageLocale: en-US
+Publisher: matchmycolor LLC
```
Comment (stephengillie) [inline manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml:7]:
```suggestion
Publisher: matchmycolor LLC
PublisherURL: https://www.matchmycolor.com/
PublisherSupportUrl: https://www.matchmycolor.com/en/support-and-services/support-desk.html
PrivacyUrl:   https://www.matchmycolor.com/en/company-overview/privacy-statement.html
```

File: manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: matchmycolorLLC.Colibri
+PackageVersion: 22.2.0.7647
+PackageLocale: en-US
+Publisher: matchmycolor LLC
+PackageName: Colibri
```
Comment (stephengillie) [inline manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml:8]:
```suggestion
PackageName: Colibri
PackageUrl: https://www.matchmycolor.com/en/landing-page-colibri-platform/landing-page-colibri-platform.html
```

File: manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: matchmycolorLLC.Colibri
+PackageVersion: 22.2.0.7647
+PackageLocale: en-US
+Publisher: matchmycolor LLC
+PublisherURL: https://www.matchmycolor.com/
```
Comment (stephengillie) [inline manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.matchmycolor.com/
```

Comment (stephengillie) [reply manifests/m/matchmycolorLLC/Colibri/22.2.0.7647/matchmycolorLLC.Colibri.locale.en-US.yaml:8]: Apologies for suggesting an incorrect change. 
> Manifest Error: All field names should be PascalCased. Field: PublisherURL

**Review by stephengillie — APPROVED** [review]

**Comment by MichaelSchae** [comment]
We would like to change the publisher to "matchmycolor" instead of "matchmycolor LLC". The package Id should then be matchmycolor.Colibri instead of matchmycolorLLC.Colibri. Is it possible to change it on the current version? Do we need to create a new pull request for this? For future versions we would like to use a different github account (github@matchmycolor.com), is that possible?
When we publish new versions with winget we would also like the have the option to retire old versions and remove them from the winget package source. How can we do that?

**Comment by stephengillie** [comment]
Hi @MichaelSchae,

> We would like to change the publisher to "matchmycolor" instead of "matchmycolor LLC". The package Id should then be matchmycolor.Colibri instead of matchmycolorLLC.Colibri. Is it possible to change it on the current version? Do we need to create a new pull request for this? 

Yes, one PR per manifest change is required, often with one manifest per version. The file path is another part of this change, so the old YAML files will be removed from `/manifests/m/matchmycolorLLC/Colibri/`, and the new YAML files added to `/manifests/m/matchmycolor/Colibri/`, losing the LLC from the path.

> For future versions we would like to use a different github account ([github@matchmycolor.com](mailto:github@matchmycolor.com)), is that possible? 

Yes, feel free to use a service account and automation to submit the above changes and/or future updates. Bot manifest submissions are generally welcome. 

> When we publish new versions with winget we would also like the have the option to retire old versions and remove them from the winget package source. How can we do that?

We have automation that monitors versions to remove old versions, among other tasks. For software where only the latest version is available, it creates one PR to add a new version, and a complementing PR to remove the old. Following that two-PR process is probably the best way to do this and stay within the one manifest change per PR requirement.

**Pull request merged**